### ISSUE TITLE
result type Float can't be cast to the desired output type Long

### ISSUE THREAD

**Issue by ahong007007** [xingyizhou/UniDet#11]
when i use 1 GPU(v100)
"python  projects/UniDet/train_net.py --config-file projects/UniDet/configs/Partitioned_COI_R50_2x.yaml --num-gpus 1",batch=16, there is no problem.

When I use two GPUs( 2x v100), an error occurred

python  projects/UniDet/train_net.py --config-file projects/UniDet/configs/Partitioned_COI_R50_2x.yaml --num-gpus 1",batch=32

![image](https://user-images.githubusercontent.com/22077027/133750054-ad995eb4-3d99-4486-ae1a-c307ffd517d3.png)

How can I solve this problem? Thank you！

**Comment by chandrakanth-gudavalli** [comment]
any update on this? I am also facing the same problem

**Comment by chandrakanth-gudavalli** [comment]
Changing "values = torch.stack(values, dim=0)" to "values = torch.stack(values, dim=0).float()" has fixed the issue.
